services/personal/service.go

```diff
@@ -0,0 +1,51 @@
+package personal
+
+import (
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/ethapi"
+	"github.com/ethereum/go-ethereum/node"
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+// Make sure that Service implements node.Service interface.
+var _ node.Service = (*Service)(nil)
+
+// Service represents out own implementation of personal sign operations.
+type Service struct {
+	am *accounts.Manager
+}
+
+// New returns a new Service.
+func New(am *accounts.Manager) *Service {
+	return &Service{am}
+}
+
+// Protocols returns a new protocols list. In this case, there are none.
+func (s *Service) Protocols() []p2p.Protocol {
+	return []p2p.Protocol{}
+}
+
+// APIs returns a list of new APIs.
+func (s *Service) APIs() []rpc.API {
+	return []rpc.API{
+		{
+			Namespace: "personal",
+			Version:   "1.0",
+			Service:   ethapi.NewLimitedPersonalAPI(s.am),
+			Public:    false,
```

Comment:
any reason not to make limited api public?

Reply:
I tried to make it the same as ETH and LES when they expose these APIs, so we are consistent there.

Reply:
i am not sure if it matters to be consistent, our api is "safe" because we use double-step approval and original api is not

Reply:
In this case, `false` means only the fact that it must be whitelisted. `true` will enable it if modules are not explicitly specified.